app/src/main/java/emu/skyline/preference/ProfilePicturePreference.kt

```diff
@@ -59,14 +61,26 @@ class ProfilePicturePreference @JvmOverloads constructor(context : Context, attr
         summaryProvider = SummaryProvider<ProfilePicturePreference> { preference ->
             Uri.decode(preference.getPersistedString("No picture selected"))
         }
+        updatePreview()
     }
 
     override fun onClick() = pickMedia.launch(PickVisualMediaRequest(ActivityResultContracts.PickVisualMedia.ImageOnly))
 
+    private fun updatePreview() {
+        var drawable: BitmapDrawable? = null
+        if (File(profilePicture).exists()) {
+            val preview = BitmapFactory.decodeFile(profilePicture)
+            if (null != preview) {
```

Comment:
I find the choice of ordering here quite funny, but we generally would do it like `preview != null`.

Reply:
Yes, checking for `null` there is completely fine. I just found it funny that `null` came first in the statement.

Reply:
It's called a yoda condition. A lot of developers use it to avoid accidental assignment. I adapted it back when it was a popular way to solve misleading exceptions in a String.equals() comparison. It never gained enough traction to become the norm for Android, though.